samples/aws/lambda/sqs/SQSLambda_0.1/AwsLambda.SQSTrigger/Function.cs

```diff
@@ -17,7 +17,7 @@ public Function()
     #region FunctionHandler
     public async Task FunctionHandler(SQSEvent evnt, ILambdaContext context)
     {
-        using (var cancellationTokenSource = new CancellationTokenSource(context.RemainingTime.Subtract(DefaultRemainingTimeGracePeriod)))
```

Comment:
@DavidBoike any reasons this was removed? When we created the sample we deliberately added this to show how the execution can be canceled with alignment with the lambda context

Reply:
FYI I'm not strictly opposed to remove this. We had a debate whether we are violating the SRP of a sample or not. At the end we added it. I'm more curious to hear if it was intentional and if yes why

Reply:
That was totally my bad. I did that to try to get the web app API to work, and forgot to revert that before committing.